flatpak/manifest_tool/flutter_ops.py

```
@@ -170,39 +170,141 @@ def normalize_lotti_env(
+
+        # Only modify the lotti module
+        if module.get("name") != "lotti":
             continue
-        build_opts = module.setdefault("build-options", {})
-        build_args = build_opts.get("build-args")
-        if isinstance(build_args, list):
-            if "--share=network" not in build_args:
-                build_args.insert(0, "--share=network")
+
+        build_commands = module.get("build-commands", [])
+        module_name = module.get("name", "unnamed")
```

Ruff (0.13.1)

Local variable `module_name` is assigned to but never used

Remove assignment to unused variable `module_name`

(F841)